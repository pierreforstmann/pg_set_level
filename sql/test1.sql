--
-- test1.sql
--
create extension pg_set_level;
--
show work_mem;
set work_mem='1GB';
show work_mem;
--
show search_path;
set search_path='';
show search_path;
--
show datestyle;
set datestyle='dmy';
show datestyle;
--
show log_statement;
set log_statement='all';
show log_statement;
--
show log_min_messages;
set log_min_messages=debug1;
-- output only in instance log
show log_min_messages;
--
-- not possible to test FATAL because breaks connection with PG
