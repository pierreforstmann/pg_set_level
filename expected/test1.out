--
-- test1.sql
--
create extension pg_set_level;
--
show work_mem;
 work_mem 
----------
 4MB
(1 row)

set work_mem='1GB';
ERROR:  pg_set_level: set work_mem='1GB';
show work_mem;
 work_mem 
----------
 4MB
(1 row)

--
show search_path;
   search_path   
-----------------
 "$user", public
(1 row)

set search_path='';
WARNING:  pg_set_level: set search_path='';
show search_path;
 search_path 
-------------
 ""
(1 row)

--
show datestyle;
   DateStyle   
---------------
 Postgres, MDY
(1 row)

set datestyle='dmy';
NOTICE:  pg_set_level: set datestyle='dmy';
show datestyle;
   DateStyle   
---------------
 Postgres, DMY
(1 row)

--
show log_statement;
 log_statement 
---------------
 none
(1 row)

set log_statement='all';
INFO:  pg_set_level: set log_statement='all';
show log_statement;
 log_statement 
---------------
 all
(1 row)

--
show log_min_messages;
 log_min_messages 
------------------
 warning
(1 row)

set log_min_messages=debug1;
-- output only in instance log
show log_min_messages;
 log_min_messages 
------------------
 debug1
(1 row)

--
-- not possible to test FATAL because breaks connection with PG
